app/layout.tsx

```diff
@@ -16,10 +17,9 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <head>
-        <link rel="icon" href="/favicon.ico" />
-      </head>
-      <body className={inter.className}>{children}</body>
+      
+    
+      <body className={montserrat.className}>{children}</body>
```

Comment:
Add `Head` section again because it contains favicon icon of `modular_ui`
```html
<head>
        <link rel="icon" href="/favicon.ico" />
</head>
```